### PR TITLE
LP-1985 BUG System is allowing user to create Organization

### PR DESCRIPTION
**Pull request by qasimgulzar**
### Description

[LP-1985](https://philanthropyu.atlassian.net/browse/LP-1985)

[philu-edx-theme PR](https://github.com/philanthropy-u/philu-edx-theme/pull/1246)

### CODE REVIEW

File: lms/envs/common.py [hunk 1]
```diff
@@ -3564,3 +3564,6 @@ def _make_locale_paths(settings):
 # CDN LINK
 
 CDN_LINK = "https://static.philanthropyu.org/"
+
+# PHILU Configuration Parameters.
+PHILU_DEFAULT_TERM_SEARCH_LENGTH = 2
```
Comment (monofal) [inline lms/envs/common.py:3569]: is this specifically related to organization? If so how about we rename this constant with something like `PHILU_ORGANIZATION_SEARCH_LENGTH `. Also this is related to on boarding don't you think this constant should be inside onboarding app. Thoughts?

Comment (qasimgulzar) [reply lms/envs/common.py:3569]: @monofal we can use it anywhere we want, as there may be other places where we want to use term length.

Comment (ahmed-faraz46) [inline lms/envs/common.py:3569]: 1. Let's please not define it in common.py because this is not something generic It is specific to org search. common.py is a core file. We can define a constant either in the same function or max in constants file in onboarding app.
2. Let's please not append PHILU with any name, reason is already discussed alot. and let's make it specific instead of generic search term something like `ORG_SEARCH_TERM_LENGTH`, `ORG_SEARCH_MIN_QUERY_LENGTH`

File: lms/djangoapps/onboarding/models.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 register(User, app=__package__, table_name='auth_historicaluser')
 register(UserProfile, table_name='auth_historicaluserprofile')
 
+models.CharField.register_lookup(models.functions.Length, 'length')
```
Comment (ahmed-faraz46) [inline lms/djangoapps/onboarding/models.py:26]: How about to directly do this at some generic place so that we can use this everywhere?

Comment (ahmed-faraz46) [reply lms/djangoapps/onboarding/models.py:26]: Don't have any suggestion about the place, but we can figure it out in a call.

Comment (qasimgulzar) [reply lms/djangoapps/onboarding/models.py:26]: @philanthropy-u/reviewers please suggest something
